### ISSUE TITLE
GR Lynchee does not support bootloader

### ISSUE THREAD

**Issue by teetak01** [ARMmbed/mbed-os-example-pelion#99]
```
Building project mbed-os-example-pelion-GR_LYCHEE-ARM-TCP (GR_LYCHEE, ARMC6)
Scan: mbed-os-example-pelion
Using ROM regions bootloader, header, application in this build.
  Region bootloader: size 0xc000, offset 0x18000000
  Region header: size 0x70, offset 0x18010000
  Region application: size 0x7efc00, offset 0x18010400
Configuration error: Bootloader not supported on this target. RAM start not found in targets.json.
```

**Comment by JanneKiiskila** [comment]
The bootloader support is in place, but unfortunately only in the Mbed OS master at this stage.
@adbridge  @0xc0170  - we need the Mbed OS 5.15.1 patch release with those changes included.
- https://github.com/ARMmbed/mbed-os/pull/12223

**Comment by ciarmcom** [comment]
*Internal Jira reference: https://jira.arm.com/browse/IOTPART-9035*

**Comment by JanneKiiskila** [comment]
Seem the changes did not make Mbed OS 5.15.1. :-(

**Comment by teetak01** [comment]
Fixed as part of Mbed OS 5.15.2 

#127
